src/translations/en.json

```
@@ -1605,6 +1605,10 @@
           "enabled_delay_confirm": "The enabled entities will be added to Home Assistant in {delay} seconds",
           "enabled_restart_confirm": "Restart Home Assistant to finish enabling the entities",
           "hidden_explanation": "Hidden entities will not be included in auto-populated dashboards or when their area, device or label is referenced. Their history is still tracked and you can still interact with them with actions.",
+          "record_label": "Record",
+          "record_description": "Control whether this entity's state history is recorded by the Home Assistant Recorder.",
```

We should drop the article when treating Home Assistant Recorder as a name:

```suggestion
          "record_description": "Control whether this entity's state history is recorded by Home Assistant Recorder.",
```